2021-02-24 12:31:09,430 INFO    MainThread:14713 [wandb_setup.py:_flush():70] setting env: {}
2021-02-24 12:31:09,430 INFO    MainThread:14713 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-02-24 12:31:09,430 INFO    MainThread:14713 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-24 12:31:09,431 INFO    MainThread:14713 [wandb_setup.py:_flush():70] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f6b743db340>
2021-02-24 12:31:09,431 INFO    MainThread:14713 [wandb_init.py:_log_setup():319] Logging user logs to /home/johannbs/Desktop/IceCube_GNN/wandb/run-20210224_123109-w4d1b0zo/logs/debug.log
2021-02-24 12:31:09,431 INFO    MainThread:14713 [wandb_init.py:_log_setup():320] Logging internal logs to /home/johannbs/Desktop/IceCube_GNN/wandb/run-20210224_123109-w4d1b0zo/logs/debug-internal.log
2021-02-24 12:31:09,431 INFO    MainThread:14713 [wandb_init.py:init():351] calling init triggers
2021-02-24 12:31:09,431 INFO    MainThread:14713 [wandb_init.py:init():356] wandb.init called with sweep_config: {}
config: {}
2021-02-24 12:31:09,432 INFO    MainThread:14713 [wandb_init.py:init():404] starting backend
2021-02-24 12:31:09,443 INFO    MainThread:14713 [backend.py:ensure_launched():81] starting backend process...
2021-02-24 12:31:09,450 INFO    MainThread:14713 [backend.py:ensure_launched():85] started backend process with pid: 14729
2021-02-24 12:31:09,451 INFO    MainThread:14713 [wandb_init.py:init():413] backend started and connected
2021-02-24 12:31:09,452 INFO    MainThread:14713 [wandb_init.py:init():436] updated telemetry
2021-02-24 12:31:09,453 INFO    MainThread:14713 [wandb_init.py:init():459] communicating current version
2021-02-24 12:31:10,205 INFO    MainThread:14713 [wandb_init.py:init():464] got version response upgrade_message: "wandb version 0.10.20 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-02-24 12:31:10,205 INFO    MainThread:14713 [wandb_init.py:init():472] communicating run to backend with 30 second timeout
2021-02-24 12:31:10,401 INFO    MainThread:14713 [wandb_init.py:init():495] starting run threads in backend
2021-02-24 12:31:12,011 INFO    MainThread:14713 [wandb_run.py:_console_start():1411] atexit reg
2021-02-24 12:31:12,011 INFO    MainThread:14713 [wandb_run.py:_redirect():1274] redirect: SettingsConsole.WRAP
2021-02-24 12:31:12,012 INFO    MainThread:14713 [wandb_run.py:_redirect():1309] Wrapping output streams.
2021-02-24 12:31:12,012 INFO    MainThread:14713 [wandb_run.py:_redirect():1325] Redirects installed.
2021-02-24 12:31:12,012 INFO    MainThread:14713 [wandb_init.py:init():518] run started, returning control to user process
2021-02-24 12:31:12,639 INFO    MainThread:14713 [wandb_config.py:__setitem__():142] config set hidden_states = 64 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f6b6c696b20>>
2021-02-24 12:31:12,639 INFO    MainThread:14713 [wandb_run.py:_config_callback():663] config_cb hidden_states 64 None
2021-02-24 12:31:12,639 INFO    MainThread:14713 [wandb_config.py:__setitem__():142] config set forward = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f6b6c696b20>>
2021-02-24 12:31:12,639 INFO    MainThread:14713 [wandb_run.py:_config_callback():663] config_cb forward False None
2021-02-24 12:31:12,639 INFO    MainThread:14713 [wandb_config.py:__setitem__():142] config set dropout = 0.25 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f6b6c696b20>>
2021-02-24 12:31:12,640 INFO    MainThread:14713 [wandb_run.py:_config_callback():663] config_cb dropout 0.25 None
2021-02-24 12:31:12,640 INFO    MainThread:14713 [wandb_config.py:__setitem__():142] config set learning_rate = 0.0001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f6b6c696b20>>
2021-02-24 12:31:12,640 INFO    MainThread:14713 [wandb_run.py:_config_callback():663] config_cb learning_rate 0.0001 None
2021-02-24 12:31:12,640 INFO    MainThread:14713 [wandb_config.py:__setitem__():142] config set batch_size = 1024 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f6b6c696b20>>
2021-02-24 12:31:12,640 INFO    MainThread:14713 [wandb_run.py:_config_callback():663] config_cb batch_size 1024 None
2021-02-24 12:31:12,641 INFO    MainThread:14713 [wandb_config.py:__setitem__():142] config set loss_func = likelihood_covariant_unitvectors - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f6b6c696b20>>
2021-02-24 12:31:12,641 INFO    MainThread:14713 [wandb_run.py:_config_callback():663] config_cb loss_func likelihood_covariant_unitvectors None
2021-02-24 12:31:12,641 INFO    MainThread:14713 [wandb_config.py:__setitem__():142] config set n_neighbors = 6 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f6b6c696b20>>
2021-02-24 12:31:12,641 INFO    MainThread:14713 [wandb_run.py:_config_callback():663] config_cb n_neighbors 6 None
2021-02-24 12:34:24,504 INFO    MainThread:14713 [wandb_init.py:_pause_backend():240] pausing backend
2021-02-24 12:35:21,238 INFO    MainThread:14713 [wandb_init.py:_resume_backend():245] resuming backend
2021-02-24 12:35:21,239 INFO    MainThread:14713 [wandb_init.py:_pause_backend():240] pausing backend
2021-02-24 12:35:21,240 INFO    MainThread:14713 [wandb_run.py:_atexit_cleanup():1381] got exitcode: 0
2021-02-24 12:35:21,240 INFO    MainThread:14713 [wandb_run.py:_restore():1353] restore
2021-02-24 12:35:21,469 INFO    MainThread:14713 [wandb_run.py:_wait_for_finish():1504] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-02-24 12:35:23,475 INFO    MainThread:14713 [wandb_run.py:_wait_for_finish():1504] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 43971
  total_bytes: 43971
}

2021-02-24 12:35:24,768 INFO    MainThread:14713 [wandb_run.py:_show_files():1726] logging synced files
